### PR TITLE
Add Educator's Guide and improve moving forward page.

### PR DESCRIPTION
**Pull request by kb5**
Add a guide to download for educators and also improve moving forward page.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -34,6 +34,10 @@ Please view our `User's Guide <_static/SongMatch.pdf>`_ for detailed information
 
 :download:`Download User's Guide <_static/SongMatch.pdf>`
 
+Also, for some ideas on how to pitch the game to children, check out our `Educator's Guide <_static/Educator_Guide.pdf>`_
```
Comment (gbroques) [inline docs/index.rst]: Can you add a period at the end of this sentence please?

File: docs/pages/moving_forward.rst [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Moving Forward
 ==============
 
+Cozmo Limitations
+-----------------
+
+During the course of development on this project, we have noticed a few limitations that can affect future additions.
```
Comment (gbroques) [inline docs/pages/moving_forward.rst]: Can you add a blank line after this? That should fix the entire intro paragraph from being bolded.

Comment (gbroques) [reply docs/pages/moving_forward.rst]: Also you cold **bold** the `Three cube limitation:` and `Lighting:` by wrapping them in asterisks `*`.

**Comment by kb5** [comment]
The changes were made. I also noticed the Educators link had the pdf extension and made it look funny.

**Pull request merged**